src/showplanner/PISModal.tsx

```diff
@@ -8,6 +8,8 @@ function DevWarning() {
   if (process.env.REACT_APP_PRODUCTION === "true") {
     return null;
   }
+  const wsUrl =
+    process.env.REACT_APP_MYRADIO_NONAPI_BASE + "/MyRadio/webstudio";
```

Comment:
I presume this is the redirect thing? It feels a bit weird to have u.o.u/myradio/MyRadio/webstudio

Reply:
Yep, it is (see https://ury.org.uk/myradio/MyRadio/webstudio/?ref=WebStudio)

Reply:
That just looks weird :/

Reply:
But whatever.